src/visitors/rules/custom/parameterAllOf.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable class-methods-use-this */
+
+class parameterAllOf {
+  constructor(config) {
+    this.config = { ...config };
+    switch (this.config.level) {
+      case 'warning':
+        this.config.level = 3;
+        break;
+      case 'error':
+      default:
+        this.config.level = 4;
+        break;
+    }
+  }
+
+  get rule() {
+    return 'parameterAllOf';
```

Comment:
Maybe use static field instead of a getter everywhere? Is there any reason go use getter?

Reply:
no reason, was used initially to get it dynamically based on path, now completely redundant

Reply:
ok, let's update it via separate PR later

Reply:
https://github.com/Redocly/openapi-cli/issues/62